docs/runtime-roadmap.md

```diff
@@ -217,7 +217,7 @@ Enso has a concept of _extension methods_. These are methods that are _not_
 defined "alongside" the type (in the same compilation unit). Currently, we have
 no way to define methods that are _not_ extensions on builtin types without
 defining them in Java. This is awkward, and leads to a poor experience for both
-developers of Enso, and the users (where there is a special case rule rule for
+developers of Enso, and the users (where there is a special case rule for
```

Comment:
Surely there's more to remove? Like... most of this section!

Reply:
Yes, but I'd like to remove it once #181499077 is out